### DISCUSSION TITLE
Is this green line Fugitive or some other Git related plugin? Also, how to turn it off?

### DISCUSSION THREAD

**Discussion by hholst80** [tpope/vim-fugitive#2107] [Q&A] [answered]
Hi guys!

I recently moved over to using Fugitive from git-gutter. I also included 100 other things from importing the nvim-starterkit set. I don't even know all the stuff in there, so I don't know for sure if this is Fugitive or something else.

![image](https://user-images.githubusercontent.com/6200749/213931890-b0fac481-fe4b-4f6d-b168-3fec3d7cb568.png)

1) is the green marker here in the "git gutter" column something that Fugitive produces?

2) how do I turn it off? At least not show the green line on every single line. Changed lines is OK to see, but I do not want to see a "green line = no change". That seems redundant to me.

Edit: As soon as you start thinking about something careful enough to pose a question, you risk finding out you were just dumb. The green line is a marker that the entire file is untracked. I would rather see that as part of the statusline rather than to see a green line in the first column across the entire screen. But at least I know better now exactly when this green line appears.

**Comment by tpope** [comment, marked as answer]
Fugitive doesn't offer any gutter functionality like this, full stop.
